dev/TeachingTip/TeachingTip_rs1_themeresources.xaml

```diff
@@ -68,11 +94,90 @@
     <GridLength x:Key="TeachingTipTailMargin">10</GridLength>
 
     <x:Double x:Key="TeachingTipAlternateCloseButtonSize">40</x:Double>
-    <x:Double x:Key="TeachingTipAlternateCloseButtonGlyphSize">12</x:Double>
-
+    <x:Double x:Key="TeachingTipAlternateCloseButtonGlyphSize">16</x:Double>
+    <Thickness x:Key="TeachingTipAlternateCloseButtonPadding">5</Thickness>
```

Comment:
5 seems like an odd value, shouldn't be 4?

Reply:
you're right, it's 4. Didn't take border thickness into account:)